tasking/pulp/tasking/celery_app.py

```diff
@@ -69,6 +72,51 @@ def initialize_worker(sender, instance, **kwargs):
     # Delete any potential old state
     delete_worker(sender, normal_shutdown=True)
 
+    if sender.startswith(constants.RESOURCE_MANAGER_WORKER_NAME):
+        get_resource_manager_lock(sender)
+
+
+def get_resource_manager_lock(name):
+    """
+    Tries to acquire the resource manager lock. If the lock cannot be acquired immediately, it
```

Comment:
It would be good to add a single-line docstring above this multi-line docstring. https://www.python.org/dev/peps/pep-0257/#id17

For example it could be:  `Block until the the resource manager lock is acquired.`